templates/homepage/demo.twig

```diff
@@ -1,32 +1,51 @@
 {% extends "base.twig" %}
 
 {% set page_title %}Try Rector Online{% endset %}
+{% set small_headline_design = true %}
 
 {% block main %}
-    <div class="container" id="subpage">
-        <div class="row" style="min-height: 30em">
-            <div class="col-12 col-md-6">
-                <h2 class="m-0">Input</h2>
-                {% form_theme form 'bootstrap_4_layout.html.twig' %}
-                {{ form(form) }}
-            </div>
-            <div class="col-12 col-md-6">
-                <h2 class="mt-0 mb-0">Result</h2>
-                    {% if rector_run is not null %}
-                        {% if rector_run.successful %}
-                            <pre class="language-diff"><code class="language-diff">{{ rector_run.contentDiff }}</code></pre>
-                        {% else %}
-                            <div class="alert alert-danger">
-                                {{ rector_run.errorMessage }}
-                            </div>
+    <div class="container-fluid mt-5 pl-5 pr-5" id="demo">
+        {% form_theme demo_form 'bootstrap_4_layout.html.twig' %}
+
+        <div id="rector_run_form">
+            {{ form_start(demo_form) }}
+                <div class="row">
+                    <div class="col-12 col-md-6">
+                        {{ form_row(demo_form.content) }}
+                    </div>
+                    <div class="col-12 col-md-6">
+                        {{ form_row(demo_form.config) }}
+
+                        {# @see http://symfony.com/doc/current/security/csrf.html#csrf-protection-in-login-forms #}
+                        <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}">
+
+                        <br>
+
+                        {{ form_row(demo_form.process) }}
+
+                        {% if rector_run is not null %}
+                            <p class="mt-4 pt-3">
+                                Permalink:
+                                <a href="{{ path('demo_detail', {'id': rector_run.id }) }}">{{ rector_run.id }}</a>
+                            </p>
                         {% endif %}
+                    </div>
+                </div>
 
-                        Permalink:
-                        <a href="{{ path('demo_detail', {'id': rector_run.id }) }}">{{ rector_run.id }}</a>
-                    {% else %}
-                        ...
-                    {% endif %}
-            </div>
+            {{ form_end(demo_form) }}
         </div>
+
+        {% if rector_run is not null %}
+            <h2 class="mt-0 mb-0">Result</h2>
```

Comment:
This is great! I hated when there was title `Result` when i did not even submitted the form 😄